config/default_records/product_catalogues/013_lcm_triomics_catalogue.wip.yml

```diff
@@ -0,0 +1,3 @@
+---
+LCM Triomics WGS and EMSeq:
```

Comment:
rename to LCM Triomics (see limber PR comment)

Reply:
This affects the grouping of Submission Templates in Limber, e.g., dropdown control in Create Submission page or the listing in Create Batch Submission pages. Setting the group name to LCM Triomics makes sense. I have now changed the product catalogue name.